assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/index.stories.js

```diff
@@ -217,6 +218,37 @@ ReadyWithActivateAnalyticsCTA.scenario = {
 	delay: 3000,
 };
 
+export const ReadyWithCompleteAnalyticsActivationCTA = Template.bind( {} );
+ReadyWithCompleteAnalyticsActivationCTA.storyName =
+	'Ready with Complete Analytics Activation CTA';
+ReadyWithCompleteAnalyticsActivationCTA.args = {
+	setupRegistry: ( registry ) => {
+		provideModules( registry, [
+			{
+				active: true,
+				connected: true,
+				slug: 'search-console',
+			},
+			{
+				active: true,
+				connected: false,
+				slug: 'analytics',
+			},
+		] );
+
+		provideSearchConsoleMockReport( registry, searchConsoleArgs );
+		provideModuleRegistrations( registry );
+	},
+};
+ReadyWithCompleteAnalyticsActivationCTA.parameters = {
+	features: [ 'zeroDataStates' ],
+};
+ReadyWithCompleteAnalyticsActivationCTA.scenario = {
+	label:
+		'SearchConsole/SearchFunnelWidget/ReadyWithCompleteAnalyticsActivationCTA',
+	delay: 3000,
+};
```

Comment:
Please remove for now.

Reply:
I removed the VRT tests for the AdminBar and their corresponding reference images as these were not added for any version of the CTA - and it is likely this CTA will be modified very soon. The reason I didn't remove the new `SearchFunnelWidget` and the `WPDashboard` tests was because they have VRTs for the `ActivateAnalyticsCTA` already. So just wanted to maintain consistency. Also, the `WPDashboard` VRTs were named at the folder level, so I renamed the existing VRTs (3) and added the new `CompleteActivationCTA` tests as well (3). I can get rid of the `delays` - I copied them over from the `ActivateAnalyticsCTA` story and thought this might be needed to due to other report data loading too for surrounding widgets?

Reply:
> The reason I didn't remove the new `SearchFunnelWidget` and the `WPDashboard` tests was because they have VRTs for the `ActivateAnalyticsCTA` already. So just wanted to maintain consistency.

@jimmymadon Thanks for clarifying – that makes sense, but we don't usually add scenarios for each variant of a component. In this case, there is a rather small difference between this story and the existing inactive variant as the only observable difference would be the wording. It's something we definitely want a Storybook story for, but doesn't need an added scenario.